api/api_public_klay.go

```diff
@@ -70,3 +75,61 @@ func (s *PublicKlayAPI) Syncing() (interface{}, error) {
 		"knownStates":   hexutil.Uint64(progress.KnownStates),
 	}, nil
 }
+
+// EncodeAccountKey gets an account key of JSON format and returns RLP encoded bytes of the key.
+func (s *PublicKlayAPI) EncodeAccountKey(accKey accountkey.AccountKeyJSON) (hexutil.Bytes, error) {
+	buffer := new(bytes.Buffer)
+	if accKey.KeyType == nil {
+		return nil, errors.New("key type is not specified")
+	}
+	key, err := accountkey.NewAccountKey(*accKey.KeyType)
+	if err != nil {
+		return nil, err
+	}
+	if err := json.Unmarshal(accKey.Key, key); err != nil {
+		return nil, err
+	}
+	// Invalidate zero values of threshold and weight to prevent users' mistake
+	// JSON unmarshalling sets zero for those values if they are not exist on JSON input
+	if err := checkAccountKeyZeroValues(key); err != nil {
```

Comment:
Is it difficult to use `key.CheckInstallable(blockNumber`)?

Reply:
The function looks similar, but do different behavior. 
For example, the function always returns false for the `AccountKeyFail` type.